whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -489,7 +498,7 @@ private static Remark createRemark(final CIString key, final AbuseContact abuseC
     }
 
     private static boolean hasDescriptionsOrRemarks(final RpslObject rpslObject) {
-        return !rpslObject.getValuesForAttribute(AttributeType.DESCR).isEmpty() || !rpslObject.getValuesForAttribute(AttributeType.REMARKS).isEmpty();
+        return !rpslObject.getValuesForAttribute(DESCR).isEmpty() || !rpslObject.getValuesForAttribute(REMARKS).isEmpty();
```

Comment:
This takes multiple attributes so you can pass both descr and remarks in single call

Reply:
I remove that method, it is only called from one place and It is clear reading the code what is doing...it is no needed to create a new method for it